state/processor.go

```diff
@@ -112,12 +112,15 @@ func (tp *TransactionProcessor) ApplyTransactions(layer LayerID, transactions Tr
 	defer tp.mu.Unlock()
 	failed := tp.Process(tp.randomSort(txs), tp.coalesceTransactionsBySender(txs))
 	newHash, err := tp.globalState.Commit(false)
-	tp.Log.Info("new state root for layer %v is %x", layer, newHash)
+
 	if err != nil {
 		tp.Log.Error("db write error %v", err)
 		return failed, err
 	}
 
+	tp.Log.Info("new state root for layer %v is %x", layer, newHash)
+	tp.Log.With().Info("new state", log.Uint32("layer id", uint32(layer)), log.String("root hash", newHash.String()))
```

Comment:
`layer id` and `root hash` will be used as json keys. it is less recommended that they have space. replace with `_`